ustriage/task.py

```diff
@@ -136,12 +136,12 @@ def short_title(self):
     def get_flags(self):
         """Get flags representing the status of the task."""
         flags = ''
-        if self.subscribed:
-            flags += '*'
-        if self.last_activity_ours:
-            flags += '+'
-        if self.AGE and self.date_last_updated > self.AGE:
+        flags += '*' if self.subscribed else ' '
+        flags += '+' if self.last_activity_ours else ' '
+        if (self.AGE and self.date_last_updated > self.AGE):
             flags += 'U'
+        else:
+            flags += ' '
```

Comment:
For consistency might be better to stick with the same if else block style:

`flags += 'U' if (self.AGE and self.date_last_updated > self.AGE) else ' '
`

Reply:
I had this initially, it breaks the >79 rule and thereby tox and our CI - that is the reason I had to fall back to this.
I could only use '\' as a continuation line, which I thought to be worse than the full if as it starts a new style (continuation lines) which we didn't have so far.